features/AssetCategory.feature

```diff
@@ -284,4 +283,102 @@ Feature: AssetCategory
     Then The document "engine-ayse":"asset-category":"littleTruck" content match:
       | parent | null |
 
+  Scenario: Use metadata enum type and link with assetCategory
+    When I successfully execute the action "device-manager/assetCategory":"create" with args:
+      | engineId  | "engine-ayse"  |
+      | body.name | "coloredTruck" |
+    When I successfully execute the action "device-manager/metadata":"create" with args:
+      | engineId       | "engine-ayse"          |
+      | body.name      | "color"                |
+      | body.valueType | "string"               |
+      | body.valueList | ["red","blue","green"] |
+      | body.mandatory | true                   |
+    When I successfully execute the action "device-manager/assetCategory":"linkMetadata" with args:
+      | engineId   | "engine-ayse"  |
+      | _id        | "coloredTruck" |
+      | metadataId | "color"        |
+    When I successfully execute the action "device-manager/assetCategory":"unlinkMetadata" with args:
+      | engineId   | "engine-ayse"  |
+      | _id        | "coloredTruck" |
+      | metadataId | "color"        |
+    When I successfully execute the action "device-manager/assetCategory":"linkMetadata" with args:
```

Comment:
You could test the content of the document after linking the metadata with the value `red` to ensure everything went well

Reply:
Done